src/ssh.c

```diff
@@ -1718,7 +1718,7 @@ int wolfSSH_KDF(byte hashId, byte keyId,
 {
     WLOG(WS_LOG_DEBUG, "Entering wolfSSH_KDF()");
     return GenerateKey(hashId, keyId, key, keySz, k, kSz, h, hSz,
-                       sessionId, sessionIdSz);
+                       sessionId, sessionIdSz, 1);
```

Comment:
Magic number of 1 here, lets comment it or use a named variable/define

Reply:
We're not using TRUE/FALSE as constants. I'm OK just having a bare 1 for true instead of a name.